ui/app/pages/send/send-content/add-recipient/ens-input.container.js

```diff
@@ -11,7 +11,7 @@ export default connect(
   (state) => {
     const selectedAddress = getSendTo(state)
     return {
-      network: getCurrentNetwork(state),
+      network: parseInt(getCurrentNetwork(state)).toString(),
```

Comment:
I logged in metamask, their network value is '1' instead of '0x1', I'm not entirely sure why the difference, but its ens support is expecting the network value as in decimal value in string instead of hex value in string.
This here is a quick fix for our ENS support, we might want to investigate more about what makes the difference in the long term and if there are other things broken due to this difference.